client/lib/user/shared-utils/filter-user-object.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Internal dependencies
+ */
+import { decodeEntities } from 'calypso/lib/formatting/decode-entities';
+import { getComputedAttributes } from './';
```

Comment:
I generally prefer to link siblings directly to each other, rather than go through the index. This helps to avoid circular dependencies, which generally aren't a problem, until they become one 😄

And in general, it keeps the dependency graph cleaner, which is useful when using tools like `whybundled`.

```suggestion
import { getComputedAttributes } from './get-computed-attributes';
```